UnitedIncome/components
# Link modification via addon-knobs

### Description
> Please provide a brief description of your changes below.

`addon-knobs` can be used for not so good things, so I'm removing the ability to modify the prop object for components that have a configurable link. I went this route instead of sanitizing the link as typically you can't modify these props via query strings unless you're viewing them via Storybook.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* I will provide reproduction steps privately. 

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

### Comments
> Any additional comments that will aid with the review process.

N/A.
